dags/transportation/dockless/scooter-stat.py

```diff
@@ -159,6 +134,4 @@ def email_callback(**kwargs):
     dag=dag,
 )
 
-task1.set_downstream(set_xcom)
-task2.set_downstream(set_xcom)
 email_task.set_upstream(set_xcom)
```

Comment:
rewrite as `set_xcom > email_task`? also perhaps rename as `computer_stats` and `send_email`?

Reply:
Yeah, that's clearer.